app/templates/index.html

```diff
@@ -10,10 +10,9 @@
           Du bist hier durch Zufall oder mit Absicht auf der Adressenvergabe
           der <em><a href="https://berlin.freifunk.net/">Berliner
               Freifunker_innen</a></em>
-          gelandet. Falls du Hilfe brauchst bei der Einrichtung deines Freifunk-
-          Routers folge am besten der
-          <em><a href="https://wiki.freifunk.net/Berlin:Firmware:Howto">Anleitung</a></em>
-          .
+          gelandet. Falls du Hilfe bei der Einrichtung deines Freifunk-
+          Routers brauchst, folge am besten der
+          <em><a href="https://wiki.freifunk.net/Berlin:Firmware:Howto">Anleitung</a></em>.
```

Comment:
This link changed to `https://wiki.freifunk.net/Berlin:Firmware/HowTo`

Reply:
fixed